cmd/res/configuration.toml

```diff
@@ -53,23 +72,24 @@ File = ''
 
 # Driver configs
 [Driver]
+[Driver]
```

Comment:
Looks like an extra` [Driver]` section got added here which causes the service to fail on startup...

Reply:
Humm, not sure how that happened. :-(

Reply:
fixed